.github/workflows/build-and-publish-sdk.yml

```diff
@@ -28,6 +28,17 @@ jobs:
           cd workflows/build/libs/
           ls | xargs -I {} sh -c 'sha256sum "{}" > {}.sha256'
 
+      - name: 'Build Dokka API reference'
+        run: ./gradlew dokkaHtmlMultiModule
+
+      # Will push HTML files to gh-pages branch
+      - name: Deploy to GitHub Pages
+        uses: peaceiris/actions-gh-pages@v3
+        if: github.ref == 'refs/heads/master'
```

Comment:
This workflow already has a rule to run only on tag pushes. This line is redundant and should be removed